language/Portuguese_Brazil/strings.po

```diff
@@ -99,19 +99,19 @@ msgstr "Favoritos"
 
 #: src/frontend/mame/ui/utils.cpp:66 src/frontend/mame/ui/miscmenu.cpp:762
 msgid "BIOS"
-msgstr ""
+msgstr "BIOS"
 
 #: src/frontend/mame/ui/utils.cpp:67
 msgid "Not BIOS"
-msgstr ""
+msgstr "Sem BIOS"
 
 #: src/frontend/mame/ui/utils.cpp:68 src/frontend/mame/ui/utils.cpp:84
 msgid "Parents"
-msgstr ""
+msgstr "Pai"
```

Comment:
MAME's parent/clone distinction is rather arbitrary, and "original" would be a poor choice of word.  The parent is generally the newest version for the most widespread release region.  In many cases, this is not the original - it's often an English translation for general distribution in Europe or Asia, or for US-developed games (e.g. from Midway) a bug-fixed release distributed as a ROM update to operators.